index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+   
+  </head>
+
+  <body>
+   <header id="title">Wangari Maathai</header>
+     
+   <main id="main">
+    <figure id="img-div">
+      <img id="image" src="https://i0.wp.com/thefootloosescribbler.com/wp-content/uploads/2018/09/wangari-2-e1537930415767.png?fit=600%2C653" alt="Woman activist" style="width:100%">
+      <figcaption id="img-caption">Wangari Maathai, Kenyan Activist</figcaption>
+    </figure>
+
+    <p id="tribute-info">Wangari Maathai, in full Wangari Muta Maathai, (born April 1, 1940, Nyeri, Kenya—died September 25, 2011, Nairobi), Kenyan politician and environmental activist who was awarded the 2004 Nobel Prize for Peace, 
+      becoming the first Black African woman to win a Nobel Prize. Her work was often considered both unwelcome and subversive in her own country, 
+      where her outspokenness constituted stepping far outside traditional gender roles.</p>
+
+      <a id="tribute-link" href="https://www.britannica.com/biography/Wangari-Maathai" 
+      target="_blank">
+      Click here for more information
+    </a>
+     </main>
```

Comment:
Please change the "main" tag to a "section" tag in lines 13 and 27 as well as give it a class rather than an id because this is your container used for your code, as id is mainly used to identify unique elements of code rather than applying to every element under a class in your CSS